From 41f234908bbeea317abe94c86353d83c3c6a0bcd Mon Sep 17 00:00:00 2001
From: Michael Krummsdorf <michael.krummsdorf@tq-group.com>
Date: Wed, 17 Jan 2018 12:42:21 +0100
Subject: [PATCH 049/188] ARM: imx: revert 'temporarily remove CONFIG_SOC_FSL
 from LS1021A'

Original commit: cfd074ad860016d3aafd28a06b03d575d17aac0a

Re-enable that symbol in awareness of the breakage in allmodconfig,
because we need it for EHCI_FSL.

Signed-off-by: Michael Krummsdorf <michael.krummsdorf@tq-group.com>
---
 arch/arm/mach-imx/Kconfig | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/arch/arm/mach-imx/Kconfig b/arch/arm/mach-imx/Kconfig
index 7693cbdf1a9f..df2db68824fb 100644
--- a/arch/arm/mach-imx/Kconfig
+++ b/arch/arm/mach-imx/Kconfig
@@ -595,10 +595,14 @@ config SOC_IMX7D
 	help
 		This enables support for Freescale i.MX7 Dual processor.
 
+config FSL_SOC
+	bool
+
 config SOC_LS1021A
 	bool "Freescale LS1021A support"
 	select ARM_GIC
 	select HAVE_ARM_ARCH_TIMER
+	select FSL_SOC
 	select ZONE_DMA if ARM_LPAE
 	help
 	  This enables support for Freescale LS1021A processor.
